karamel-core/src/main/java/se/kth/karamel/backend/running/model/tasks/VendorCookbookTask.java

```diff
@@ -45,16 +46,28 @@ public VendorCookbookTask(MachineRuntime machine, ClusterStats clusterStats, Tas
   @Override
   public List<ShellCommand> getCommands() throws IOException {
     String cookbookPath = githubRepoName;
+    String githubuser = ClusterService.getInstance().getCommonContext().getGithubUsername();    
     if (subCookbookName != null && !subCookbookName.isEmpty()) {
       cookbookPath += Settings.SLASH + subCookbookName;
     }
     if (commands == null) {
+      String httpProxy = System.getenv("http.proxy");
+      if (httpProxy == null) {
+        httpProxy = "";
+      }
+      String httpsProxy = System.getenv("https.proxy");      
+      if (httpsProxy == null) {
+        httpsProxy = "";
+      }
```

Comment:
Did you test this one? It doesn't look correct. If you assume that users have setup the environment variable correctly then then name should be `http_proxy` and `https_proxy`.

User could have also specified the proxy in the System properties (see: https://github.com/karamelchef/karamel/commit/ddeeb336bafc20d08ff30b805dfc19ca553e6c73) - but even in that case the property name is wrong.

Reply:
The problem here is that http_proxy_host and http_proxy_port are what is needed by Java. But wget and github need the full URL - https.proxy or http.proxy. I need these env variables to pass them to wget and git in the first setup recipes karamel runs (hard-coded apt-get.sh and clone-....sh files)